FAQ - Add feedback email to qr_test

heinezen opened the pull request:
Resolves https://github.com/corona-warn-app/cwa-website/issues/749#issuecomment-771633933

One thing that could be improved in a later PR is that "unusually long waiting period" is not defined. I could not find enough reliable information on how fast the tests are reported under normal circumstances. However, we should add this info to this entry if it becomes available.

MikeMcC399 commented:
@heinezen 
This is a good addition to the FAQ article! Thank you for following up! I looked at your changes and it all looks good to me.

I doubt that there is a definite maximum duration for the delivery of test results unless there is a Federal Service Level Agreement for this. Pragmatically a person being tested could ask at the time of testing:
a) if the result will be delivered by the CWA app
b) how long it is expected to take before a result is delivered

There are probably lots of factors including the day of the week, backlog of tests, which lab is analyzing the sample, etc.  Nevertheless, if there is some objective figure to add to the FAQ, to replace the vague "a few days" that would help to filter the reports coming in.

GisoSchroederSAP commented:
All, just a quick insight here: With an up-to-date technical equipment, the standard PCR testing lasts about 4 to 6 hours in the lab. However, including logistics and documentation the labs report a time frame of 48 hours to including entry processing, analysis and test result reporting to the LIS (laboratory information system). So, after two days we become disquiet. 
Fortunately, if anything works fine and result gets transmitted digitally, the current average is 22...28 hours, which is the next day.
Home, this is valuable for the discussion.

MikeMcC399 commented:
@GisoSchroederSAP 
Thank you very much for providing some concrete figures! That helps a lot to get the right expectations.

If you monitor the Verification Server on an on-going basis would it be possible to give a figure for the time-lapse between the generation of a Registration token (after the user has scanned their QR code) and the delivery of a result to the app (after the lab has posted the result and the app has polled the server)? It would be good to have the reference time for which 90% of all test results are delivered to the app.

If the user knows the 90% value and the average value for the time-lapse between scanning a QR code and getting a result, they could compare to see if they are over the average and over the 90% value. If they are over the 90% value then it is quite likely that something has gone wrong and they would be justified in reporting an issue. If the average time has not yet expired for the user in their situation they would know that they should be patient and just wait a bit longer.

(See https://github.com/corona-warn-app/cwa-verification-server/blob/master/docs/architecture-overview.md for Verification Server documentation.)

dsarkar commented:
DE

![image](https://user-images.githubusercontent.com/1409741/106741382-e2ca7000-661b-11eb-8e19-c5b97d7929f7.png)

dsarkar commented:
EN

![image](https://user-images.githubusercontent.com/1409741/106741443-fbd32100-661b-11eb-9218-fdce6eb0ce06.png)

MikeMcC399 commented:
@dsarkar 
Do you want to replace "unusually long waiting periods" before merging?

Also, do you know why the [check ](https://github.com/corona-warn-app/cwa-website/pull/832/checks) is failing?

dsarkar commented:
@MikeMcC399 

> Also, do you know why the check is failing?

Right now, no clue.
We have several PR here, e.g. FAQ-Portugal, when I merge master into the working-branch, it fails.

dsarkar commented:
@MikeMcC399 

regarding, 
> unusually long waiting periods

yes, we can change that before merging. ...

dsarkar commented:
In the case of the PR FAQ-Portugal, it tells me where the conflict is, but there shouldn't be a conflict, since this line hadn't been changed. The only PR on the website yesterday was the faq-statistics.

MikeMcC399 commented:
@dsarkar 
> > unusually long waiting periods
> 
> yes, we can change that before merging. ...

So I suggest adding the guideline of 2 days from @GisoSchroederSAP 

For example:
"You can report cases of long waiting periods (up to two days is considered normal) ..."

"Sie können Fälle von ungewöhnlich langen Wartezeiten (bis zwei Tage gilt als normal) ..."

The check fail may just be a temporary glitch. It seems to be saying that a file is missing, but the ones it references should be there. 🤷‍♂️

heinezen commented:
@GisoSchroederSAP @MikeMcC399 

Thanks Giso! I added this info to the FAQ entry. The entry now calls for reports to the mail address if reporting the test result takes 3+ days.

I have also added a more general FAQ entry that references the average and maximum waiting time for the test result in the Use/Anwendung section: 

> **When do I receive my test result in the app?**
> Currently, analyzing the test in the laboratory and reporting the test result to the Corona-Warn-App servers takes 22-28 hours on average. However, the waiting time can be up to 48 hours depending on the commissioned laboratory. In most cases, you will receive your result the next day or the day after. If your result is not available in the app after 3 days, please follow the advice in FAQ entry: I have scanned a QR code, but the test result could not be retrieved for days.

german

> **Wann erhalte ich mein Testergebnis in der App?**
> Bis der Test im Labor untersucht and das Testergebnis an die Server der Corona-Warn-App gemeldet wird vergehen derzeit im Durchschnitt 22-28 Stunden. Jedoch kann die Wartezeit je nach beauftragtem Labor bis zu 48 Stunden betragen. In den allermeisten Fällen erhalten Sie Ihr Ergebnis also am nächsten oder übernächsten Tag. Falls Ihr Testergebnis nach 3 Tagen noch nicht in der App abrufbar ist, beachten Sie bitte die Hinweise im FAQ-Eintrag: Ich habe einen QR Code eingescannt, aber das Testergebnis konnte über Tage nicht abgerufen werden.

MikeMcC399 commented on src/data/faq_de.json:
```diff
@@ -557,6 +558,14 @@
+                        "anchor": "result_time",
+                        "active": true,
+                        "textblock": [
+                            "Bis der Test im Labor untersucht and das Testergebnis an die Server der Corona-Warn-App gemeldet wird vergehen derzeit im Durchschnitt 22-28 Stunden. Jedoch kann die Wartezeit je nach beauftragtem Labor bis zu 48 Stunden betragen. In den allermeisten Fällen erhalten Sie Ihr Ergebnis also am nächsten oder übernächsten Tag. Falls Ihr Testergebnis nach 3 Tagen noch nicht ...
```
wird vergehen => wird, vergehen (missing comma)

MikeMcC399 reviewed (changes requested):
One punctuation suggestion for you.

dsarkar commented:
DE

![image](https://user-images.githubusercontent.com/1409741/107025545-e2f87600-67a9-11eb-885d-b20515dd9e4e.png)

dsarkar commented:
EN

![image](https://user-images.githubusercontent.com/1409741/107026648-4fc04000-67ab-11eb-88d5-08b6464df15d.png)

dsarkar commented:
DE EN new Entry

![image](https://user-images.githubusercontent.com/1409741/107027044-e4c33900-67ab-11eb-899f-51e84941914f.png)

MikeMcC399 commented on src/data/faq_de.json:
```diff
@@ -557,6 +558,14 @@
+                        "anchor": "result_time",
+                        "active": true,
+                        "textblock": [
+                            "Bis der Test im Labor untersucht and das Testergebnis an die Server der Corona-Warn-App gemeldet wird, vergehen derzeit im Durchschnitt 22-28 Stunden. Jedoch kann die Wartezeit je nach beauftragtem Labor bis zu 48 Stunden betragen. In den allermeisten Fällen erhalten Sie Ihr Ergebnis also am nächsten oder übernächsten Tag. Falls Ihr Testergebnis nach 3 Tagen noch nich ...
```
and => und

heinezen replied:
fixed

MikeMcC399 reviewed (changes requested):
English word "and" in German text 🙂 !

heinezen commented:
Merged!
